test/test_operations.py

```diff
@@ -2642,14 +2642,14 @@ def test_api(self):
 
     ctx = torch_xla._XLAC.lowering.LoweringContext("MyCustomName")
     ctx.build([result])
-    hlo = ctx.hlo()
```

Comment:
I think it's still good to preserve this call to test it doesn't throw an exception. Maybe `_ = ctx.hlo()` to be more explicit.